opds_import.py

```diff
@@ -671,15 +681,32 @@ def summary_to_linkdata(detail):
         return kwargs_meta
 
     @classmethod
-    def rights_uri_for_entry(cls, entry):
+    def rights_uri(cls, rights_string):
         """Determine the URI that best encapsulates the rights status of
         the downloads associated with this book.
+        """
+        return RightsStatus.rights_uri_from_string(rights_string)
+
+    @classmethod
+    def rights_uri_from_feedparser_entry(cls, entry):
+        """Extract a rights URI from a parsed feedparser entry.
 
-        :return: A URI
+        :return: A rights URI.
         """
         rights = entry.get('rights', "")
-        return RightsStatus.rights_uri_from_string(rights)
+        return cls.rights_uri(rights)
+        
+    @classmethod
+    def rights_uri_from_entry_tag(cls, entry):
+        """Extract a rights string from an lxml <entry> tag.
 
+        :return: A rights URI.
+        """
+        for name in ('atom:rights', 'rights'):
+            rights = OPDSXMLParser._xpath1(entry, 'rights')
```

Comment:
Yes, and the fact that everything still works makes me think line 705 isn't necessary.